### PR TITLE
Add PHP 8 images

### PR DESCRIPTION
**Pull request by frankverhoeven**
Drops PHP 7.3 images as these aren't and won't be used.
Drops Memcached 
Upgrades redis to 5.3.3 as required for PHP 8

Note that the PHP 8 images do not (yet) have Imagick on board, because there isn't a compatible release yet. See https://github.com/Imagick/imagick/issues/358 for more info. I don't think that that should be a show-stopper, because we probably won't need it in a PHP 8 container anytime soon anyway. The alternative would be to build from source (master).

### CODE REVIEW

File: fpm-7.2-alpine/Dockerfile [hunk 1]
```diff
@@ -40,32 +38,28 @@ RUN chmod u+x /usr/local/bin/docker-php-ext-get \
     # pre-fetch pecl extensions
  && cd / \
  && docker-php-source extract \
- && docker-php-ext-get imagick 3.4.4 \
- && docker-php-ext-get memcached 3.1.5 \
- && if [ "7.2" != "7.0" ]; then docker-php-ext-get pcov 1.0.6;fi \
+ && if [ "7.2" != "8.0" ]; then echo docker-php-ext-get imagick 3.4.4 ;fi \
```
Comment (digibeuk) [inline fpm-7.2-alpine/Dockerfile:41]: this wil always result in true :smile:

Comment (frankverhoeven) [reply fpm-7.2-alpine/Dockerfile:41]: It's generated from the template ;)

Comment (digibeuk) [reply fpm-7.2-alpine/Dockerfile:41]: so confused right now :confused:

Comment (frankverhoeven) [reply fpm-7.2-alpine/Dockerfile:41]: See https://github.com/MyOnlineStore/docker-php-dev/pull/9/files#diff-79e47e40e79d9731a46d49fe6b1b2e6a9e3ed637df8c855acf5e864ca98f3655R90

**Review by digibeuk — APPROVED** [review]

**Pull request merged**